src/main/java/com/github/joelittlejohn/embedmongo/StartMojo.java

```diff
@@ -151,12 +158,14 @@ public void executeStart() throws MojoExecutionException, MojoFailureException {
         try {
 
             final ICommandLinePostProcessor commandLinePostProcessor;
-            if (authEnabled) {
+            final List<String> mongodArgs = this.createMongodArgsList(); 
+            if (!mongodArgs.isEmpty()) {
```

Comment:
No need to do this `if`. In case the list ist empty we just pass an empty args list onto the command line.